### PR TITLE
Update auto-notify bot for array-api label

### PR DESCRIPTION
**Pull request by kmaehashi**
Adding array-api notification to @asmeurer :tada:
https://github.com/cupy/cupy/pull/5698#issuecomment-915733273

The workflow is tested here: https://github.com/kmaehashi/sandbox/issues/12

### CODE REVIEW

File: .github/workflows/auto-cc.yml [hunk 1]
```diff
@@ -10,23 +10,27 @@ jobs:
   auto-notify:
     if: |
         github.repository_owner == 'cupy' &&
-        github.event.label.name == 'hip'
+        contains('|hip|array-api|', github.event.label.name)
```
Comment (kmaehashi) [inline .github/workflows/auto-cc.yml:13]: Unfortunately `env` is not accessible from within `if` condition.

File: .github/workflows/auto-cc.yml [hunk 2]
```diff
@@ -10,23 +10,27 @@ jobs:
...
+    env:
+      NOTIFY_TARGETS: '{"hip": "@amathews-amd", "array-api": "@asmeurer"}'
```
Comment (leofang) [inline .github/workflows/auto-cc.yml:15]: Would this work? 😛
```suggestion
      NOTIFY_TARGETS: '{"hip": "@amathews-amd", "array-api": "@asmeurer @leofang"}'
```

Comment (kmaehashi) [reply .github/workflows/auto-cc.yml:15]: Sure! I thought you are watching everything 😉

**Comment by kmaehashi** [comment]
Let me merge this and check it's working.

**Pull request merged**